AppCenter/AppCenter/Internals/Model/CommonSchema/MSCSData.m

```diff
@@ -35,13 +35,13 @@ - (BOOL)isEqual:(id)object {
 
 - (instancetype)initWithCoder:(NSCoder *)coder {
   if ((self = [super init])) {
-    _properties = [coder decodeObjectForKey:kMSDataProperties];
+    _properties = [coder decodeObject];
   }
   return self;
 }
 
 - (void)encodeWithCoder:(NSCoder *)coder {
-  [coder encodeObject:self.properties forKey:kMSDataProperties];
+  [coder encodeRootObject:self.properties];
```

Comment:
Just a thought: Is this something we might want to do for other implementations of `encodeWithCoder:`, too?

Reply:
This case is special, there is no key associated to the object, so from this perspective it's just the root object.

Reply:
I see. Maybe we can add a comment to point out the special case?